_quarto.yml

```diff
@@ -226,19 +223,32 @@ website:
             - text: "Controlling for Page Size"
               href: "/layouts/arrange/index.html#controlling-for-page-width-and-height"
 
-    - id: docs
+    - id: get-started
       style: "floating"
-      collapse-level: 2
+      collapse-level: 1
       align: left
       contents:
-        - section: "Essentials"
+        - get-started/overview.qmd
+        - get-started/user-interfaces.qmd
+        - get-started/install-create-run.qmd
+        - get-started/debug.qmd
+        - section: "Deploy"
           contents:
-            - docs/overview.qmd
-            - docs/user-interfaces.qmd
-        - section: "Workflow"
+            - text: "Overview"
+              href: get-started/deploy.qmd
+            - get-started/deploy-cloud.qmd
+            - get-started/deploy-on-prem.qmd
+            - get-started/shinylive.qmd
+        - section: "Tutorials"
           contents:
-            - docs/install-create-run.qmd
-            - docs/debug.qmd
+            - text: "Get Started with Shiny Express"
+              href: tutorials/intro-express/1-welcome.qmd
```

Comment:
I like that we have all of these pieces together in one place. I think now it's worth going through these to make sure that they all make sense together. Now that they're here are they well organized, do the sections work well together, etc?

When I compare this page with [pydantic's Get Started](https://docs.pydantic.dev/latest/), ours still feels disorganized.